BETSE 0.9.2 bumped.

leycec opened the pull request:
This commit bumps conda-forge hosting to the most recent stable release: **BETSE 0.9.2** (_Luckiest Levin_).

<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

leycec committed:
BETSE 0.9.2 bumped.

This commit bumps conda-forge hosting to the most recent stable release:
BETSE 0.9.2 (Luckiest Levin).

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

leycec committed:
Tarball domain updated.

The PyPI-hosted domain of this package's tarball has been updated from
the now seemingly dead pypi.io to files.pythonhosted.org. This is what
redirects are for, infrastructure maintainers. *sigh*

leycec committed:
Tarball domain updated x 2.

The PyPI-hosted domain of this package's tarball has been updated from
the seemingly inaccessible files.pythonhosted.org to pypi.org. This is
what redirects are for, infrastructure maintainers. *sigh*

leycec commented:
@conda-forge-admin, please rerender

conda-forge-admin committed:
MNT: Re-rendered with conda-build 3.17.7, conda-smithy 3.2.9, and conda-forge-pinning 2019.01.21

leycec commented:
Resounding huzzahs for automated rerendering. :clap:

leycec commented:
I'm here to bump feedstocks and chew bubblegum, and I'm all out of bubblegum.
